scripts/actions/mobskills/rising_swell.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+--  Rising Swell
+--  Description: Strips all players in an area of effect of all equipment.
+--  Type: Magical Enfeebling
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    for i = xi.slot.MAIN, xi.slot.BACK do
+        target:unequipItem(i)
+    end
+
+    skill:setMsg(xi.msg.basic.NONE)
+    return 0
+end
```

Comment:
We could use a `target:isPC()` check in there aswell

Reply:
Good point on this, do we play nicely if the target is a pet or trust?

Reply:
Done! If the entity is not a PC then the `unequipItem` call is a noop but it doesn't really hurt to the check here as well.